bpfd/src/lib.rs

```diff
@@ -12,6 +12,7 @@ mod static_program;
 mod utils;
 
 use anyhow::{bail, Context};
+//use aya::programs::ProgramError;
```

Comment:
Nit remove this comment

Reply:
Will do.  I wanted to go back and figure out why that wasn't needed anymore first.